test/security/WSSecurity.js

```diff
@@ -39,8 +39,8 @@ describe('WSSecurity', function() {
   });
 
   it('should insert a WSSecurity when postProcess is called', function() {
-    var username = 'myUser';
-    var password = 'myPass';
+    var username = 'special&<<>>Characters]]>username</wsse:Username>';
+    var password = 'even&>><<"moreSpecial]]>password';
```

Comment:
could you please remove the `]]` characters from any of the two variables (either `username` or `password`) so that the test ensures that it'll work without them too? 👍